app/main.py

```
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
+    if len(parts) != 3 or parts[0] != "cp" or parts[1] == parts[2]:
+        return
+
+    try:
+        with open(parts[1], "r") as src, open(parts[2], "w") as dst:
+            dst.write(src.read())
+    except FileNotFoundError:
+        pass
```

The current implementation silently passes when a `FileNotFoundError` occurs. It might be more helpful to log an error message or raise an exception to inform the user that the source file does not exist. This would improve the usability of the function by providing more informative feedback.